packages/delivery-fetch/README.md

```diff
@@ -0,0 +1,7 @@
+# @bugsnag/delivery-fetch
+
+This package contains the fetch implementation of the Bugsnag notifier delivery system for JavaScript.
```

Comment:
We're using the new casing in all new readmes etc&hellip; — not the license though!

```suggestion
This package contains the fetch implementation of the BugSnag notifier delivery system for JavaScript.
```